internal/server/store.go

```diff
@@ -220,11 +221,27 @@ func (s *Server) buildConfigSnapshot(name string) error {
 		if u.Password != "" {
 			user.Password = u.Password
 		}
-		users = append(users, user)
+
+		if u.Account != "" {
+			if account, ok := accounts[u.Account]; ok {
```

Comment:
Minor logic change could simplify.  If `!ok`, create the data structures, then always append users after falling through the if - make sense?

Reply:
Updated